pkg/controller/ehpa/predict.go

```diff
@@ -160,6 +160,10 @@ func IsPredictionEnabled(ehpa *autoscalingapi.EffectiveHorizontalPodAutoscaler)
 	return ehpa.Spec.Prediction != nil && ehpa.Spec.Prediction.PredictionWindowSeconds != nil && ehpa.Spec.Prediction.PredictionAlgorithm != nil
 }
 
+func IsCronScaleEnabled(ehpa *autoscalingapi.EffectiveHorizontalPodAutoscaler) bool {
```

Comment:
s/IsCronScaleEnabled/CronEnabled/g

Reply:
> s/IsCronScaleEnabled/CronEnabled/g

I keep it same style with the IsPredictionEnabled, so the IsPredictionEnabled need to rename to PredictionEnabled?

Reply:
ok, how about IsCronEnabled.